api/bases/dataplane.openstack.org_openstackdataplanenodesets.yaml

```diff
@@ -1910,6 +1910,10 @@ spec:
                   - type
                   type: object
                 type: array
+              configChanged:
```

Comment:
This is not implemented in https://github.com/openstack-k8s-operators/dataplane-operator/pull/491